src/TLuaInterpreter.cpp

```diff
@@ -12734,6 +12731,11 @@ void TLuaInterpreter::logError( std::string & e, const QString & name, const QSt
 
     if( mpHost->mpConsole && mpHost->mEchoLuaErrors )
```

Comment:
There is not a need to check for a valid `mpHost->mpConsole` pointer here.  The `cTelnet` class code explicitly handles the (start-up) case where the `mpConsole` pointer may be Null - it stacks up the messages until one comes in with the pointer valid - at which point it discharges any/all stacked up messages before displaying the latest arriving one... :nerd_face: